app/css/components/modal.css

```diff
@@ -38,11 +38,34 @@
             right: 48px;
             height: 64px;
             width: 64px;
+
             & .c-icon {
                 height: 32px;
                 width: 32px;
                 filter: var(--filter-FFFFFF);
             }
+
+            &:hover {
+                @apply bg-wildBlueYonder;
+            }
+
+            &:active,
+            &:focus {
+                @apply bg-crayola;
+            }
+
+            &:focus {
+                @apply border-2 border-purpleDarker3;
+                @apply outline outline-3 outline-lightLavender;
```

Comment:
IIRC, we currently don't use outlines anywhere as they don't go round like borders. Where are you seeing the outline in the design? Could you link me to figma pls?